ros/src/twist_controller/twist_controller.py

```diff
@@ -10,77 +12,112 @@
 
 
 class Controller(object):
-    def __init__(self, vehicle_mass, fuel_capacity, brake_deadband, decel_limit, accel_limit, wheel_radius,
-                 wheel_base, steer_ratio, max_lat_accel, max_steer_angle, kp, ki, kd):
-        rospy.logdebug("wheel_base: {}\tsteer_ratio: {}\tmax_lat_accel: {}\tmax_steer_angle: {}\n".format(wheel_base, steer_ratio, max_lat_accel, max_steer_angle))
-
-        self.wheel_radius = wheel_radius
-        self.brake_deadband = brake_deadband
-        self.deadband_torque = BrakeCmd.TORQUE_MAX * self.brake_deadband
-        self.total_mass = vehicle_mass + fuel_capacity * GAS_DENSITY
-        self.decel_limit = decel_limit
+    def __init__(self, *args, **kwargs):
+        self._wheel_radius = kwargs['wheel_radius']
+        self._brake_deadband = kwargs['brake_deadband']
+        self._accel_limit = kwargs['accel_limit']
+        self._decel_limit = kwargs['decel_limit']
+	self._max_throttle = kwargs['max_throttle']
```

Comment:
I am a big fan of code improvements, too - but large changes like these make it harder to review a PR that's supposed to be small. Maybe next time :)